### PR TITLE
Fix bugs found during testing

### PR DESCRIPTION
**Pull request by rafsalas19**
- Bug fixes to versioning of perf test install
- added additional check for topo files
- adjust set point for A100

### CODE REVIEW

File: customTests/custom-test-setup.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ function install_perf_test(){
 	fi
 
 	pushd ${EXE_DIR}
-	perftest_dir="perftest-${VERSION}"
+	perftest_dir="perftest-4.5"
```
Comment (yakovdyadkin) [inline customTests/custom-test-setup.sh]: Is it possible to grab 1st field before "-" of `VERSION`? Hardcoded version is used here while line 45 still uses variable

Comment (rafsalas19) [reply customTests/custom-test-setup.sh]: let me check

File: customTests/custom-test-setup.sh [hunk 2]
```diff
@@ -82,9 +82,18 @@ if lspci | grep -iq NVIDIA ; then
...
-	#copy necessary files
-	mkdir -p /opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles/
-	cp $SRC_DIR/topofiles/* /opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles/
+	#copy topo files
+	TOPO_PATH=/opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles
+
+	mkdir -p $TOPO_PATH
+
+	if [ ! -e "$TOPO_PATH/ndv4-topo.xml" ]; then
+		cp $SRC_DIR/topofiles/ndv4-topo.xml /opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles/
```
Comment (yakovdyadkin) [inline customTests/custom-test-setup.sh]: Use `TOPO_PATH`?

File: customTests/custom-test-setup.sh [hunk 3]
```diff
@@ -82,9 +82,18 @@ if lspci | grep -iq NVIDIA ; then
...
+	fi
+
+	if [ ! -e "$TOPO_PATH/ndv5-topo.xml" ]; then
+		cp $SRC_DIR/topofiles/ndv5-topo.xml /opt/azurehpc/test/azurehpc-health-checks/customTests/topofiles/
```
Comment (yakovdyadkin) [inline customTests/custom-test-setup.sh]: Same as above

File: customTests/custom-test-setup.sh (hunk 2 above)
Comment (rafsalas19) [reply customTests/custom-test-setup.sh]: will fix

Comment (moes1) [inline customTests/custom-test-setup.sh:91]: Use TOPO_PATH variable here?

Comment (rafsalas19) [reply customTests/custom-test-setup.sh]: fixed

File: customTests/custom-test-setup.sh (hunk 1 above)
Comment (rafsalas19) [reply customTests/custom-test-setup.sh]: fixed

**Review by moes1 — APPROVED** [review]

**Review by yakovdyadkin — APPROVED** [review]

**Pull request merged**